sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -59,10 +59,19 @@ def get_ancillary(instrument, descriptor):
         Download path of calibration file.
     """
     imap_data_access.config["DATA_DIR"] = EFS_BASE_PATH
-    # TODO: this only takes the first file. Sort out what calibration files are needed.
-    calibration_file = imap_data_access.query(
-        table="ancillary", instrument=instrument, descriptor=descriptor
-    )[0]
+    calibration_files = imap_data_access.query(
+        table="ancillary",
+        instrument=instrument,
+        descriptor=descriptor,
+        version="latest",
+    )
+
+    if not calibration_files:
+        raise FileNotFoundError(
+            f"No calibration file found for {instrument=}, {descriptor=}"
+        )
+
+    calibration_file = calibration_files[0]
```

Comment:
```suggestion
    try:
        calibration_files = imap_data_access.query(
            table="ancillary",
            instrument=instrument,
            descriptor=descriptor,
            version="latest",
        )[0]

    Except FileNotFoundError:
        print(f"No calibration file found for {instrument=}, {descriptor=}")
```

Just a suggestion to simplify this a bit, but not a big deal either way.